### PR TITLE
ENG-462

### PR DESCRIPTION
**Pull request by egh**
Combined PR of https://jira.plos.org/jira/browse/ENG-462 and https://jira.plos.org/jira/browse/ENG-343

This PR includes changes to the `enqueue.py` script to add some new features:
- the "final" migration step for rhino (see the confluence page below for more information)
- the "final" migration step for lemur
- the tracking of the last ids processed, so the scripts can be used in an incremental migration/syncing process
- deploy of this as cron jobs in GKE

Steps to deploy this in GKE:
- `cd gcsmigrate`
- `docker build . -t gcr.io/plos-mgmt/eng-462`
- `docker push gcr.io/plos-mgmt/eng-462:latest`
- `cd k8s/dev && kubectl apply -k .`

This is currently deployed and working in dev GKE.

See also https://confluence.plos.org/confluence/display/ENG/Journals+GCP+Migration

### CODE REVIEW

File: gcsmigrate/k8s/base/final_migrate_job.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: lemursync-final
+spec:
+  schedule: "5,15,25,35,45,55 * * * *"
```
Comment (mjvaldez) [inline gcsmigrate/k8s/base/final_migrate_job.yaml:6]: just fyi you can also write this like
5-59/10 * * * *

File: gcsmigrate/k8s/base/container.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+apiVersion: apps/v1
```
Comment (mjvaldez) [inline gcsmigrate/k8s/base/container.yaml:1]: Is this container just for debugging?

File: gcsmigrate/k8s/base/final_migrate_job.yaml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+                - -c
+                - "python enqueue.py final_migrate_rhino && python enqueue.py final_migrate_lemur"
+              volumeMounts:
+                - mountPath: /lemursync
+                  name: lemursync-data
+              envFrom:
+                - configMapRef:
+                    name: lemursync-configmap
+                - secretRef:
+                    name: lemursync
```
Comment (mjvaldez) [inline gcsmigrate/k8s/base/final_migrate_job.yaml:26]: a little pedantic but I usually call this something like 'lemursync-secrets' so you know it's a secret when referring to it later.

**Review by mjvaldez — DISMISSED** [review]
just a couple of nits to pick but otherwise LGTM

File: gcsmigrate/k8s/base/container.yaml (hunk 2 above)
Comment (egh) [reply gcsmigrate/k8s/base/container.yaml:1]: Yep!

File: gcsmigrate/k8s/base/final_migrate_job.yaml (hunk 1 above)
Comment (egh) [reply gcsmigrate/k8s/base/final_migrate_job.yaml:6]: I wanted this to run after the migrate job, every 10 minutes on the 5. Will that work?

**Pull request merged**